packages/components/socialFeed/NewsFeed/NewsFeed.tsx

```diff
@@ -38,8 +35,10 @@ export const NewsFeed: React.FC<NewsFeedProps> = ({
 }) => {
   const { data, isFetching, refetch, hasNextPage, fetchNextPage, isLoading } =
     useFetchFeed(req);
-  const isLoadingValue = useSharedValue(false);
-  const isGoingUp = useSharedValue(false);
+  const isLoadingValue = useMemo(
```

Comment:
We can assign directly,  I think useMemo does not help much in this case and make the code longer :D

Reply:
And my suggestion, the var name can be `isReady` =>
`const isReady = !(isFetching || isLoading)`